_opcodes/op116.html

```diff
@@ -10,6 +10,16 @@
 iwd2: 1
 pst: 1
 ---
-Removes invisibility from the targeted creature(s).<br/>
-This effect ignores durations attributed to it.<br/>
+Removes invisibility from the targeted creature(s).
+
+{% capture note %}
+This effect ignores durations attributed to it.
```

Comment:
Nobody writes effects from scratch, it's much easier to ship the file or modify whatever it is attached to. A duration is always set, since it's an non-optional field and the point of the note is that it doesn't matter at all.